low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -262,7 +255,11 @@ def validate(self):
         pass
 
     def process(self):
-        Cmd._call_script(f"{self._script_dir}/{self.script}", self._args)
+        manifest_support_bundle = f"{self._script_dir}/{self.script} {' '.join(self._args)}"
+        output, error, returncode = SimpleProcess(manifest_support_bundle).run()
+        if returncode != 0:
+            sys.stderr.write("error: %s\n\n" % str(error))
```

Comment:
Avoid print and exit from the function. You should return exception. main function which is calling this will handle this. Example...
 
```
def main(argv: dict):
    try:
        command = Cmd.get_command(desc, argv[1:])
        command.process()

    except Exception as e:
        sys.stderr.write("error: %s\n\n" % str(e))
        sys.stderr.write("%s\n" % traceback.format_exc())
        Cmd.usage(argv[0])
        return errno.EINVAL
```

Reply:
Done.